README.md

```diff
@@ -61,6 +61,12 @@ operations on the wallets. The package has excellent documentation for a smooth
 
 ***starknet simpleCall***
 
+1. Rename the ".env.template" file located at the root of the "examples" folder to ".env"
```

Comment:
We don't need to repeat all the instructions, they are already located in the README of each example. The user can check then with the "Check here for more details" button.
Let's remove this and the other two too